src/main/java/org/gridsuite/modification/server/entities/ModificationEntity.java

```diff
@@ -50,28 +50,53 @@ public class ModificationEntity {
     @Column(name = "modifications_order")
     private int modificationsOrder;
 
+    @Column(name = "message_type")
+    private String messageType;
+
+    @Column(name = "message_values")
+    private String messageValues;
```

Comment:
Do we really need to store modifications metadata in the database ? wouldn't it be better the generate them on the fly when we explicitly ask for them ?